src/chain.rs

```diff
@@ -67,7 +75,15 @@ pub trait PevmChain: Debug {
         txs: &BlockTransactions<Self::Transaction>,
         tx_results: &[PevmTxExecutionResult],
     ) -> B256;
+
+    /// Build Self::Transaction type from Alloy's transaction
```

Comment:
Let's move this and the test changes to a PR to merge first.
Also put this below `id`.

Reply:
See https://github.com/risechain/pevm/pull/344